db/external_sst_file_ingestion_job.cc

```diff
@@ -291,6 +291,12 @@ void ExternalSstFileIngestionJob::Cleanup(const Status& status) {
     // We failed to add the files to the database
     // remove all the files we copied
     for (IngestedFileInfo& f : files_to_ingest_) {
+      // If internal_file_path is empty str, we have reached the first file
+      // that has not been linked/copied successfully, thus no need to iterate
+      // over the rest of files_to_ingest_.
+      if (f.internal_file_path.empty()) {
+        break;
```

Comment:
Will `continue` generate the same outcome? If that is the case, maybe do it as it is more correct in a self-contained way.

Reply:
Yes they will lead to the same outcome. Using `break` needs a stronger assumption than `continue`, but I do not see difference in terms of performance. I will update this logic.